pkg/agent/testdata/AKSUbuntu1604+AKSCustomCloud/CustomData

```diff
@@ -35,6 +35,8 @@ write_files:
 
 
 
+
+
```

Comment:
is it possible to get rid of this two empty lines?

Reply:
removed